recipes/libproperties/config.yml

```diff
@@ -0,0 +1,3 @@
+versions:
+  "1.1.2":
```

Comment:
I don't think there is a `1.1.2` version.
You can use `cci.20181001` for using the current latest git commit at 1st of October 2018.

Reply:
But I do not want to change the recipe for each release. Are there some way to do this?

Reply:
You don't have to change the recipe if you write the build script in such a way that it will work for the next libproperties release.
For the next release, you just have to open a pr adding the new version + download url + sha256 checksum.